src/test/java/org/mule/extensions/internal/execution/JavaArgumentTransformationTestCase.java

```diff
@@ -118,7 +119,23 @@ public void invokeStaticWithListOfMapsAndStringsTransformed() throws Exception {
     assertThat(payload.getValue(), is(28));
   }
 
+  @Test
+  public void invokeStaticWithNullParameter() throws Exception {
+    TypedValue<String> payload = flowRunner("invokeStaticWithNullParameter")
+        .run()
+        .getMessage()
+        .getPayload();
+    assertNull(payload.getValue());
+  }
 
+  @Test
+  public void invokeStaticWithMapWithListValuesAndOneIsNull() throws Exception {
+    TypedValue<String> payload = flowRunner("invokeStaticWithMapWithListValuesAndOneIsNull")
+        .run()
+        .getMessage()
+        .getPayload();
+    assertThat(payload.getValue(), is("11"));
+  }
```

Comment:
add one negative test where the parameter does not accept a null value and invocation fails